ggml/src/ggml.c

```diff
@@ -21760,7 +21904,6 @@ int ggml_cpu_has_neon(void) {
 int ggml_cpu_has_sve(void) {
 #if defined(__ARM_FEATURE_SVE)
     // TODO: Currently, SVE 256 bit is only supported.
```

Comment:
Is this `TODO` still relevant?

Reply:
Yes, we can remove this `TODO` becasue there is now a fallback path (NEON path) for Arm cpus with SVE VL not equal to 256 bits.